core/message.md

```diff
@@ -34,6 +37,7 @@ class MessageComputer:
         signature: string;
         address: string;
         version: int;
+        signer: string
```

Comment:
Correct. But when updating sdk-js we'll do it as optional, to avoid a breaking change in v13. Actually, to delay it a bit (at some point, it will be made).